help/destinations/catalog/advertising/tradedesk-emails.md

```diff
@@ -0,0 +1,122 @@
+---
+title: The Trade Desk - CRM
+description: Activate profiles to your Trade Desk account for audience targeting and suppression based on CRM data.
+---
+
+# The Trade Desk - CRM
+
+
+## Overview {#overview}
+
+This document is designed to help you activate profiles to your Trade Desk account for audience targeting and suppression based on CRM data.
+
+The Adobe Real-Time CDP requires granular identity type mappings - if and when using multiple identity mappings (cookie, iOS, Android, email, hashed email, phone, etc), multiple destinations need to be created. The Trade Desk CRM destination is meant to be used for CRM data mapping, like email or hashed email address.   
```

Comment:
Apologies, but I don't understand the purpose of this sentence: 

The Adobe Real-Time CDP requires granular identity type mappings - if and when using multiple identity mappings (cookie, iOS, Android, email, hashed email, phone, etc), multiple destinations need to be created.

Can you please rephrase or remove it? 

Also, in the overview, can we please briefly explain the difference between this new Trade Desk CRM destination and the old one in our catalog? Something like: 

The Trade Desk CRM destination is meant to be used for CRM data mapping, like email or hashed email address. The other Trade Desk destination in the Adobe catalog is meant to be used for <insert purpose here>.

Reply:
During our discussion, you had asked for a description of the differences between "The Trade Desk" destination and "The Trade Desk - CRM" destination; the purpose of the sentence was to provide context and describe the difference between the two destinations and why both destinations are used. 

The language is pulled from existing documentation that lives within the Adobe Audience Manager platform when integrating with "The Trade Desk" destination. 

Without context into the requirements of the platform, the differences between the destinations is not clear.